packages/experimental/Switch/src/__tests__/Switch.test.tsx

```diff
@@ -13,9 +16,11 @@ it('Switch Disabled', () => {
   expect(tree).toMatchSnapshot();
 });
 
-it('Simple switch rendering does not invalidate styling', () => {
-  checkRenderConsistency(() => <Switch label="Default Switch" />, 2);
-});
+// Temporary commented out to check on Animation , seems AnimatedAPI brings difference in render tree.
```

Comment:
Yeah, I think this is expected with Animated. We can remove this test.

Reply:
Removed.